components/paystack/actions/initialize-transaction.mjs

```diff
@@ -0,0 +1,46 @@
+import paystack from "../app/paystack.app.mjs";
+
+export default {
+  key: "paystack-initialize-transaction",
+  name: "Initialize Transaction",
+  description:
+    "Initializes a new transaction on Paystack. [See the documentation](https://paystack.com/docs/api/transaction/#initialize)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    paystack,
+    email: {
+      propDefinition: [paystack, "email"],
+    },
+    amount: {
+      propDefinition: [paystack, "amount"],
+    },
+    currency: {
+      propDefinition: [paystack, "currency"],
+      optional: true,
+    },
+    reference: {
+      propDefinition: [paystack, "reference"],
+      optional: true,
+    },
+    callback_url: {
+      propDefinition: [paystack, "callback_url"],
+      optional: true,
+    },
+    metadata: {
+      propDefinition: [paystack, "metadata"],
+      optional: true,
+    },
```

Comment:
These props aren't being used. Please remove them or add tem to the request.

Reply:
Hi @tkalejaiye, what I meant is that you forgot to pass these props to the initializeTransaction function. you are passing only email and amount.
And about your question, there is no way to bundle props, only individually

Reply:
Oh yeah, I understood that @luancazarine. My [latest commit](https://github.com/PipedreamHQ/pipedream/pull/10256/commits/6797cd5d3a2256778aa36cb32d9da3b0e47619f1) passes the props to the initializeTransaction function. Sorry if that wasn't clear in my last message